packages/office-ui-fabric-react/src/components/Text/Text.view.tsx

```diff
@@ -1,10 +1,11 @@
 /** @jsx withSlots */
+import * as React from 'react';
 import { withSlots, getSlots } from '../../Foundation';
 import { getNativeProps, htmlElementProperties } from '../../Utilities';
 import { ITextComponent, ITextProps, ITextSlots } from './Text.types';
 
 export const TextView: ITextComponent['view'] = props => {
-  if (!props.children) {
+  if (React.Children.count(props.children) === 0) {
```

Comment:
Can children explicitly be `null` or `undefined`? What does the code do in these cases? What should we do in this case?

Reply:
`React.Children.count` returns 0 in case `null` or `undefined` are passed. You can see that in [this codepen](https://codepen.io/khmakoto/pen/ZEYxqME?editors=0010)